sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSJUnitPlugin.scala

```diff
@@ -0,0 +1,43 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js sbt plugin        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+package org.scalajs.sbtplugin
+
+import sbt._
+import sbt.Keys._
+
+object ScalaJSJUnitPlugin extends AutoPlugin {
+  override def requires: Plugins = ScalaJSPlugin
+
+  override def projectSettings: Seq[Setting[_]] = {
+    ScalaJSPluginInternal.scalaJSAbstractSettings ++
+      ScalaJSPluginInternal.scalaJSEcosystemSettings ++
```

Comment:
Don't include those. They are already in `ScalaJSPlugin.projectSettings`. If you put them here, they will be duplicated.